choreography/src/choreography/Frabrikam.Choreography.ChoreographyService.Tests/ChoreographyControllerFixture.cs

```diff
@@ -10,28 +10,32 @@
 using Fabrikam.Choreography.ChoreographyService.Models;
 using Fabrikam.Choreography.ChoreographyService.Services;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Azure.EventGrid;
-using Microsoft.Azure.EventGrid.Models;
+using Azure.Messaging.EventGrid;
+using Azure.Messaging.EventGrid.SystemEvents;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Internal;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
 
-
 namespace Fabrikam.Choreography.ChoreographyService.Tests
 {
     [TestClass]
     public class ChoreographyControllerFixture
     {
         private static Delivery delivery = new Delivery();
-        SubscriptionValidationEventData eventValidationData = new SubscriptionValidationEventData(Guid.NewGuid().ToString(), "http://url");
+        public static object obj = Activator.CreateInstance(typeof(SubscriptionValidationEventData));
```

Comment:
For mocking and testing purposes, we should be using the [`EventGridModelFactory`](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventgrid/Azure.Messaging.EventGrid/src/Generated/EventGridModelFactory.cs) to create types not normally intended for developer creation.

Reply:
Thanks for your guidance!  It looks good using `EventGridModelFactory`.